bust-a-move-framework/src/main/java/bustamove/player/Player.java

```diff
@@ -77,8 +77,10 @@ public final String getName() {
      * @param s String value to set name to
      */
     public final void setName(final String s) {
-        Log.getInstance().log(this, "Name " + this.name + " changed to " + s);
-        this.name = s;
+        String formatted = Character.toTitleCase(s.charAt(0)) + s.substring(1);

```

Comment:
Hmm nice one, this one capitalizes all first letters of separate words instead of just the first word/name. Nice Job!